.changeset/perfect-mangos-greet.md

```diff
@@ -0,0 +1,24 @@
+---
+"@comet/admin-color-picker": major
+"@comet/admin-react-select": major
+"@comet/admin-date-time": major
+"@comet/blocks-admin": major
+"@comet/admin-icons": major
+"@comet/admin-theme": major
+"@comet/admin-rte": major
+"@comet/cms-admin": major
+"@comet/admin": major
+---
+
+Migrate to MUI v6
```

Comment:
```suggestion
Upgrade to MUI v6
```

I'd call this upgrade, since we doesn't really migrate from one thing to another